source/MaterialXTest/MaterialXGenMdl/GenMdl.cpp

```diff
@@ -177,7 +178,7 @@ TEST_CASE("GenShader: MDL Shader Generation", "[genmdl]")
 
     const mx::FilePath logPath("genmdl_mdl_generate_test.txt");
 
-    bool writeShadersToDisk = false;
+    bool writeShadersToDisk = true;
```

Comment:
This will produce code but not run the mdlc test unless specified (which is false by default).